UK/Aberdeen/Aberdeen SMR.prf

```diff
@@ -35,9 +35,9 @@ Settings	airportcoords	\..\Data\Datafiles\icao.txt
 Settings	airways	\..\Data\Datafiles\airway.txt
 Settings	performance	\..\Data\Datafiles\Performance.txt
 LastSession	server	AUTOMATIC
-LastSession	atis_url0	http://uniatis.net/atis.php?arr=$arrrwy($atisairportA)&dep=$deprwy($atisairportA)&apptype=ILS&info=$atiscodeA&metar=$metar($atisairportA)
-LastSession	atis_url1	http://uniatis.net/atis.php?arr=$arrrwy($atisairportB)&dep=$deprwy($atisairportB)&apptype=ILS&info=$atiscodeB&metar=$metar($atisairportB)
-LastSession	atis_url2	http://uniatis.net/atis.php?arr=$arrrwy($atisairportC)&dep=$deprwy($atisairportC)&apptype=ILS&info=$atiscodeC&metar=$metar($atisairportC)
-LastSession	atis_url3	http://uniatis.net/atis.php?arr=$arrrwy($atisairportD)&dep=$deprwy($atisairportD)&apptype=ILS&info=$atiscodeD&metar=$metar($atisairportD)
+LastSession	atis_url0	https://www.vatatis.nz/gen?apptype=ILS&arr=$arrrwy($atisairportA)&dep=$deprwy($atisairportA)&metar=$metar($atisairportA)&info=$atiscodeA
```

Comment:
I would prefer we keep the server generated ATIS letter, rather than the one made up by ES. That means, just remove the &info parameter.